src/App.tsx

```diff
@@ -1,18 +1,116 @@
+import React, { useState, useEffect } from 'react';
+import { Routes, Route } from 'react-router-dom';
 import { PeoplePage } from './components/PeoplePage';
 import { Navbar } from './components/Navbar';
-
+import { HomePage } from './components/homePage';
+import { PageNotFound } from './components/PageNotFound';
+import { Person, NewPerson } from './types';
+import { getPeople } from './api';
+import { FilterType } from './components/enum';
 import './App.scss';
 
-export const App = () => {
+export const App: React.FC = () => {
+  const [people, setPeople] = useState<Person[]>([]);
+  const [newPeople, setNewPeople] = useState<NewPerson[]>([]);
+  const [loading, setLoading] = useState(true);
+  const [sexFilter, setSexFilter] = useState(FilterType.All);
+  const [query, setQuery] = useState('');
+  const [isError, setIsError] = useState(false);
+
+  const sexFilterHandler = (value: FilterType) => {
+    setSexFilter(value);
+  };
+
+  const deleteQuery = () => {
+    setQuery('');
+  };
+
+  useEffect(() => {
+    const fetchPeople = async () => {
+      try {
+        const fetchedData = await getPeople();
+
+        setLoading(false);
+        setPeople(fetchedData);
+      } catch (error) {
+        // eslint-disable-next-line no-console
+        console.error('Error occurred:', error);
+        setIsError(true);
+      }
+    };
+
+    fetchPeople();
+  }, []);
+
+  useEffect(() => {
+    const updatedPeople = people.map((person, index) => {
+      return {
+        ...person,
+        index: index + 1,
+      };
+    });
+
+    setNewPeople(updatedPeople);
+  }, [people]); //
+
   return (
     <div data-cy="app">
       <Navbar />
 
       <div className="section">
         <div className="container">
-          <h1 className="title">Home Page</h1>
-          <h1 className="title">Page not found</h1>
-          <PeoplePage />
+          <Routes>
+            <Route path="*" element={<PageNotFound />} />
+            <Route path="/" element={<HomePage />} />
+            <Route
+              path="/people"
+              element={(
+                <PeoplePage
+                  people={newPeople}
+                  loading={loading}
+                  sexFilter={sexFilter}
+                  sexFilterHandler={sexFilterHandler}
+                  query={query}
+                  setQuery={setQuery}
+                  deleteQuery={deleteQuery}
+                  isError={isError}
+                />
+              )}
+            >
```

Comment:
Don't pass all props from the App component. You can get all values in the PeoplePage directly 
```suggestion
            <Route
              path="/people"
              element={<PeoplePage/>}
            >
```